src/pages/Sponsors.js

```diff
@@ -1,82 +1,119 @@
-import { Helmet } from "react-helmet";
 import Page from "../components/Page";
 
 // for testing only, not for final production
 import TestLogo from "../assets/KnightHacksLogo.svg";
-
+import Facebook from "../assets/FBLogo.png";
+import EA from "../assets/EA.png";
+import Google from "../assets/GoogleCloud.png";
+import UCF from "../assets/UCF.png";
+import TI from "../assets/TI.png";
+import Learner from "../assets/Learner.png";
+import synopsys from "../assets/synopsys.jpg";
 /**
  * @desc sponsors page; still WIP since we are waiting on sponsors
  * @author Rob
  */
 const Sponsors = () => {
-  const sponsorsIncomplete = true;
```

Comment:
please undo the deleted code! we can keep reference to it in case we don't want to show sponsors, its a very useful flag. instead change the variable thats passed to the if case so its false instead of true

Reply:
Added Back